### PR TITLE
Fix resize observer problem

### PR DESCRIPTION
**Pull request by nenonja**
## Description :sparkles:

Fix resize observer problem in testcafe: https://github.com/DevExpress/testcafe/issues/4857#issuecomment-598775956
Originally failed build is here: https://github.com/City-of-Helsinki/events-helsinki-ui/pull/180/checks?check_run_id=1701632369
## Additional notes :spiral_notepad:

### CODE REVIEW

File: testcafe/e2e/changeLanguage.ts [hunk 1]
```diff
@@ -4,9 +4,12 @@ import translationsFi from '../../src/common/translation/i18n/fi.json';
 import translationsSv from '../../src/common/translation/i18n/sv.json';
 import { header } from '../selectors/header';
 import { getPathname } from '../utils/clientUtils';
+import jsErrorHandler from '../utils/jsErrorHandler';
 import { getEnvUrl } from '../utils/settings';
 
-fixture('Landing page').page(getEnvUrl('/fi/home'));
+fixture('Landing page')
+  .clientScripts(jsErrorHandler)
```
Comment (Sartonon) [inline testcafe/e2e/changeLanguage.ts:11]: Very god tha there was solution for this. Should we add small comment why this is added? In addition to the see more link

Comment (nenonja) [reply testcafe/e2e/changeLanguage.ts:11]: added

Comment (Sartonon) [reply testcafe/e2e/changeLanguage.ts:11]: Great, for some reason I have missed this change and comment

**Review by Sartonon — APPROVED** [review]
Great!!

**Pull request merged**